src/main/java/com/ap3xtbh/raidbot/commands/util/Event.java

```diff
@@ -0,0 +1,113 @@
+package com.ap3xtbh.raidbot.commands.util;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+/**
+ * Represents an event that can be scheduled in the LFG command.
+ */
+public class Event {
+    private static final Logger log;
+
+    static {
+        log = LoggerFactory.getLogger(Event.class);
+    }
+
+    // constant strings to retrieve json file entries for events
+    private final String JSON_LONG_NAME = "long_name";
+    private final String JSON_TYPE = "type";
+    private final String JSON_PLAYER_COUNT = "player_count";
+    private final String JSON_EXACT_PLAYER_COUNT = "exact_player_count";
+    private String shortName;
+    private String longName;
+    private String type;
+    private int playerCount = 0;
+    private boolean exactPlayerCount = false;
+
+    public Event(String shortName) {
+        this.shortName = shortName;
+        load(shortName);
+    }
+
+    public String getShortName() {
+        return shortName;
+    }
+
+    public String getLongName() {
+        return longName;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public int getPlayerCount() {
+        return playerCount;
+    }
+
+    public boolean isExactPlayerCount() {
+        return exactPlayerCount;
+    }
+
+    public String getJSON_LONG_NAME() {
+        return JSON_LONG_NAME;
+    }
+
+    public String getJSON_TYPE() {
+        return JSON_TYPE;
+    }
+
+    public String getJSON_PLAYER_COUNT() {
+        return JSON_PLAYER_COUNT;
+    }
+
+    public String getJSON_EXACT_PLAYER_COUNT() {
+        return JSON_EXACT_PLAYER_COUNT;
+    }
+
+    /**
+     * Load event information from event config json.
+     * @param shortName - event shortname to load from.
+     */
+    public void load(String shortName) {
+        try {
+            JsonElement element = JsonParser.parseReader(new FileReader("event.json"));
+            JsonObject eventConfig = element.getAsJsonObject();
+            JsonObject eventAsJSON = eventConfig.getAsJsonObject(shortName);
+
+            this.longName = eventAsJSON.get(JSON_LONG_NAME).getAsString();
```

Comment:
Did we want to pull some of this repeated code into a method? Or was that part of another issue for later?

Reply:
This is to be done in #8